dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SolrQueryResponse.java

```diff
@@ -25,6 +21,8 @@
     private Map<String, List<String>> spellingSuggestionsByToken;
     private List<FacetCategory> facetCategoryList;
     private List<FacetCategory> typeFacetCategories;
+    private Map<String, String> datasetfieldFriendlyNamesBySolrField = new HashMap<>();
+    private Map<String, String> staticSolrFieldFriendlyNamesBySolrField;
```

Comment:
Shouldn't this also be initilized?

Reply:
Also should the keyword static be here since it is not static field :P?

Reply:
I'm gonna leave the name as it is. This fields are probably connected with translations on 'Publication Status' facet which changed during development of My Data page and requires fix on this new page now.
Possibly they might get deleted but that requires a closer look.